qmerge.c

```diff
@@ -934,24 +939,47 @@ pkg_merge(int level, const depend_atom *atom, const struct pkg_t *pkg)
 			size_t mlen = fread(magic, 1, sizeof(magic), mfd);
 			fclose(mfd);
 
-			if (mlen >= 2 && magic[0] == 'B' && magic[1] == 'Z') {
+			if (mlen >= 3 && magic[0] == 'B' && magic[1] == 'Z' &&
+					magic[2] == 'h') {
 				compr = "j";
-			} else if (mlen >= 4 &&
-					magic[0] == 037 && magic[1] == 0213 &&
-					magic[2] == 010 && magic[3] == 00)
-			{
+			} else if (mlen >= 2 &&
+					magic[0] == 037 && magic[1] == 0213) {
 				compr = "z";
 			} else if (mlen >= 5 &&
 					magic[1] == '7' && magic[2] == 'z' &&
-					magic[3] == 'X' && magic[4] == 'Z')
-			{
+					magic[3] == 'X' && magic[4] == 'Z') {
```

Comment:
this is a change not documented above

Reply:
I misread, nevertheless, I don't like the change because it's not at all related to this change.  If you were changing the line, I'd be ok with it.  But since I (as sole maintainer) am slowly migrating the code to a more readable (IMO) form, there is no such thing as "the rest of the file/project".

Reply:
In this case, the other lines I inserted should be reformatted as well.
(When I had to decide about their "layout", I looked at majority of the code.)

Should I update the PR? Should I re-base it?

Reply:
no, it's fine, just remove this particular change.

Reply:
that said, if you feel like it, just move the { to the next line for multi-line conditions, but I'm not going to reject this just because of some style nits.